src/pages/beta/[viewType]/programmes/[programmeSlug]/units.tsx

```diff
@@ -132,6 +134,7 @@ const UnitListingPage: NextPage<UnitListingPageProps> = ({
         subjectIconBackgroundColor={"lavender"}
         title={`${subjectTitle} ${examBoardTitle ? examBoardTitle : ""}`}
         programmeFactor={keyStageTitle}
+        isNew={viewType === "teachers-2023"}
```

Comment:
Why is this needed for this PR ?

Reply:
I made some changes to the tag promo component and noticed that on the lesson and units page it was missing for new lessons/units subject icon